config/prometheus/prometheus_role_binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+  - kind: ServiceAccount
+    name: prometheus-k8s
+    namespace: openshift-monitoring
```

Comment:
Just wonder whether we care about monitoring on k8s? in that case this should be configurable.

Reply:
we'll have to address k8s in a different task.
for that purpose we'd probably like to change the way we're using kustomize.
also, for k8s we should ease the process of installing prometheus.
but that is out of the boundaries of this PR/task.